PLIER_pathways_analysis.Rmd

```diff
@@ -0,0 +1,76 @@
+---
+title: "PLIER pathways analysis"
+output: html_notebook
+---
+
+```{r}
+meaningful_difference <- 0.2
+```
+
+### What additional pathways are identified by PLIER after doubling the sample size?
+
+Here we look for cancer pathways that are found more frequently in the data with the full sample size than in data with half the sample size.
```

Comment:
```suggestion
Here we look for oncogenic pathways that are found more frequently in the data with the full sample size than in data with half the sample size.
```

It might be even better to include a citation for where `oncogenicPathways` comes from. (I assume it's an MSigDB collection?)

Reply:
Added MsigDB C6 link